### DISCUSSION TITLE
How i can Avoid the 6mb limit limitation

### DISCUSSION THREAD

**Discussion by jesusrodrz** [dotansimha/graphql-code-generator#6430] [Q&A]
![image](https://user-images.githubusercontent.com/34176666/128408118-c7ebda97-9441-446d-b3ac-10ddaff4b048.png)

**Comment by dotansimha** [comment]
Hi
Are you loading your schema from AWS Lambda? Because it seems like AWS has a limit for 6MB response size - this has nothing to do with codegen (we don't have any limitations on file/schema size).
